docker-compose.yml

```diff
@@ -53,3 +68,62 @@ services:
     entrypoint: >
       curl -v -X POST --data-binary '{"name":"seafowl-test-bucket"}' -H "Content-Type:
       application/json" "http://fake-gcs:4443/storage/v1/b"
+
+configs:
+  dex.yaml:
```

Comment:
It's nice that this works, but since the config is not small it might make sense to actually extract it into a proper file.